index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Matee Safi - Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+
+<body>
+      <header class="nav">
+        <nav>
+            <div class="logo-container">
+                <a class="logo-text">Matee Safi</a>
+            </div>
+            <div class="menu">
+                <img id="hamburger" src="./assets/hamburger-icon.svg" alt="hamburger menu">
+            </div>
+        </nav>
+      </header>
+    <section class="headline">
+        <div class="main-line">
+            <h1 class="orange">Hey There.<br>I'm Matee</h1>
+        </div>
+        <div class="second-line">
+            <h3 class="orange">I am a Software Developer</h3>
```

Comment:
:red_circle: I think you can switch this h3 to h5 to fix the design issue as for now it doesn't follow the Figma Design.

|Your Design with h3 |Your Design with h5 | Figma Design|
|:----------------:|:----------------:|:-----------------:|
|![des2](https://user-images.githubusercontent.com/26736582/212361857-092749b8-fbc3-487e-b980-e34e1869e863.png)|![desi](https://user-images.githubusercontent.com/26736582/212360811-6e00cf80-ba2f-4e75-a04a-adb3cea89317.png)|<img src="https://user-images.githubusercontent.com/99246359/212210054-ba430b24-a4b8-44c5-b206-6abf456352ba.png"/>|

Reply:
@bienvenuushindi it's actually a <p> tag not h3.